Dapper.Tests/PerformanceTests.cs

```diff
@@ -129,49 +131,51 @@ public void Run(int iterations)
                 Try(() =>
                 {
                     var l2scontext1 = GetL2SContext(connection);
-                    tests.Add(id => l2scontext1.Posts.First(p => p.Id == id), "Linq 2 SQL");
+                    tests.Add(id => l2scontext1.Posts.First(p => p.Id > id), "Linq 2 SQL");
 
                     var l2scontext2 = GetL2SContext(connection);
-                    var compiledGetPost = CompiledQuery.Compile((Linq2Sql.DataClassesDataContext ctx, int id) => ctx.Posts.First(p => p.Id == id));
+                    var compiledGetPost = CompiledQuery.Compile((Linq2Sql.DataClassesDataContext ctx, int id) => ctx.Posts.First(p => p.Id > id));
                     tests.Add(id => compiledGetPost(l2scontext2, id), "Linq 2 SQL Compiled");
 
                     var l2scontext3 = GetL2SContext(connection);
-                    tests.Add(id => l2scontext3.ExecuteQuery<Post>("select * from Posts where Id = {0}", id).First(), "Linq 2 SQL ExecuteQuery");
+                    tests.Add(id => l2scontext3.ExecuteQuery<Post>("select * from Posts where Id > {0}", id).First(), "Linq 2 SQL ExecuteQuery");
                 }, "LINQ-to-SQL");
 #endif
 
 #if ENTITY_FRAMEWORK
+                
                 Try(() =>
                 {
                     var entityContext = new EFContext(connection);
-                    tests.Add(id => entityContext.Posts.First(p => p.Id == id), "Entity framework");
+                    tests.Add(id => entityContext.Posts.First(p => p.Id > id), "Entity framework");
+
 
+                    //var entityContext2 = new EFContext(connection);
+                    tests.Add(id => entityContext.Posts.SqlQuery("select * from Posts where Id > {0}", id).First(), "Entity framework SqlQuery");
 
-                    var entityContext2 = new EFContext(connection);
-                    tests.Add(id => entityContext2.Database.SqlQuery<Post>("select * from Posts where Id = {0}", id).First(), "Entity framework SqlQuery");
 
                     //var entityContext3 = new EFContext(connection);
-                    //tests.Add(id => entityFrameworkCompiled(entityContext3, id), "Entity framework CompiledQuery");
+                    //tests.Add(id => entityContext.Posts.FirstOrDefault(x=>x.Id==id), "Entity framework Normal");

```

Comment:
The entire point of these is *not* sharing a context, so that fetch and deserialization can be compared. This makes no sense as a test anymore if they all use the same context and are fetching from the object map.